### ISSUE TITLE
Add `section` property to action

### ISSUE THREAD

**Issue by TNorbury** [TNorbury/command_palette#21]
### Discussed in https://github.com/TNorbury/command_palette/discussions/15

<div type='discussions-op-text'>

<sup>Originally posted by **karniv00l** July  6, 2022</sup>
Hi 👋 

I think it would be nice to be able to categorize actions with something like a `section`. Like in [kbar](https://github.com/timc1/kbar).

Example:

<img width="615" alt="Screenshot 2022-07-06 at 22 07 25" src="https://user-images.githubusercontent.com/3144291/177634059-4425039e-55a3-46d4-8269-8fae5277f2e9.png">

</div>

**Comment by romanr** [comment]
Yes, especially if you use command palette for commands and search, which always the case, it needs visual separation of commands and data search results.
